tests/cli-v2.rs

```diff
@@ -1109,3 +1109,24 @@ fn target_list_ignores_unavailable_targets() {
         expect_not_stdout_ok(config, target_list, clitools::CROSS_ARCH1);
     })
 }
+
+#[test]
+fn install_with_component_and_target() {
+    setup(&|config| {
+        expect_ok(config, &["rustup", "default", "nightly"]);
+        expect_ok(
+            config,
+            &[
+                "rustup",
+                "toolchain",
+                "install",
+                "nightly",
+                "-c",
+                "clippy",
+                "-t",
+                clitools::CROSS_ARCH1,
+                "--no-self-update",
+            ],
+        );
```

Comment:
Should this test for thee presence of the component and target afterwards?

Reply:
Yeah I think it probably should.